src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -144,6 +145,12 @@ export const selectPortfolioAccountMetadata = createDeepEqualOutputSelector(
   accountMetadata => accountMetadata,
 )
 
+export const selectPortfolioAccountMetadataByAccountId = createSelector(
+  selectPortfolioAccountMetadata,
+  selectAccountIdParamFromFilter,
```

Comment:
While at it, about time we type this as an `AccountId` ?

```diff
diff --git a/src/state/slices/portfolioSlice/selectors.ts b/src/state/slices/portfolioSlice/selectors.ts
index 8966228c8..a3659332a 100644
--- a/src/state/slices/portfolioSlice/selectors.ts
+++ b/src/state/slices/portfolioSlice/selectors.ts
@@ -69,14 +69,14 @@ import {
 
 type ParamFilter = {
   assetId: AssetId
-  accountId: AccountSpecifier
+  accountId: AccountId
   accountSpecifier: string
   validatorAddress: PubKey
   supportsCosmosSdk: boolean
 }
 type OptionalParamFilter = {
   assetId: AssetId
-  accountId?: AccountSpecifier
+  accountId?: AccountId
   accountSpecifier?: string
   validatorAddress?: PubKey
   supportsCosmosSdk?: boolean
```

Before:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/17035424/188119481-53deb7d8-934c-4bf2-8ed1-ab43e9e3a2ad.png">

After:

<img width="789" alt="image" src="https://user-images.githubusercontent.com/17035424/188119527-9a9578a6-2f2d-4626-b198-f4434cdb6a47.png">